src/universal/modules/userDashboard/containers/CreditCardModal/CreditCardModalContainer.js

```diff
@@ -88,7 +93,13 @@ class CreditCardModalContainer extends Component {
         stripeToken
       };
       const {error: cashayError} = await cashay.mutate('addBilling', {variables});
-      if (cashayError) throw new SubmissionError(cashayError);
+      if (cashayError) {
+        dispatch(
+          segmentEventTrack('addStripeBilling Cashay Error', {error: cashayError._error})
```

Comment:
I'll toss the error one, but let's keep the success one. It's key for the funnel metrics.